.github/workflows/generate_and_push_deriv_ui.yml

```diff
@@ -0,0 +1,70 @@
+name: Generate and push @deriv/ui
+on:
+    workflow_dispatch:
+
+jobs:
+    generate_and_push_deriv_ui:
+        if: github.repository == 'deriv-com/deriv-components'
+        runs-on: 'ubuntu-latest'
+        steps:
+            - name: Setup node and npm
+              id: step1
+              uses: actions/setup-node@v2
+              with:
+                  node-version: '16'
+                  check-latest: true
+                  registry-url: 'https://registry.npmjs.org'
+              env:
+                  NODE_AUTH_TOKEN: ${{ secrets.NPM_ACCESS_TOKEN }}
+            - name: Checkout repo
+              id: step2
+              uses: actions/checkout@v3
+              with:
+                  fetch-depth: 1
+                  ref: main
+            - name: Bump version and create PR
+              id: step3
+              run: |
+                  package_name="@deriv/ui"
+                  # Identify with Git, these values come from the shared Deriv FE account. https://github.com/DerivFE
+                  git config --global user.name "DerivFE"
+                  git config --global user.email "80095553+DerivFE@users.noreply.github.com"
```

Comment:
TODO: Confirm if these credentials are still valid

Reply:
This is fine @adrienne-deriv